client/webserver/webserver.go

```diff
@@ -1085,7 +1085,13 @@ func writeJSON(w http.ResponseWriter, thing any, indent bool) {
 // ResponseWriter with the specified response code.
 func writeJSONWithStatus(w http.ResponseWriter, thing any, code int, indent bool) {
 	w.Header().Set("Content-Type", "application/json; charset=utf-8")
-	b, err := json.Marshal(thing)
+	var b []byte
+	var err error
+	if indent {
+		b, err = json.MarshalIndent(thing, "", "	") // tab
+	} else {
+		b, err = json.Marshal(thing)
+	}
```

Comment:
I'd be up for use removing the `indent bool` argument altogether. I don't think we're using it anywhere.

Reply:
Removed in https://github.com/decred/dcrdex/pull/2803/commits/47429e13a79293df073d491232727a51327d1380.